src/196AA0.c

```diff
@@ -240,7 +212,7 @@ INCLUDE_ASM(s32, "196AA0", func_80268E88);
 s32 check_block_input(s32 buttonMask) {
     BattleStatus* battleStatus = &gBattleStatus;
     PlayerData* playerData = &gPlayerData;
-    struct D_8029FBE0* d8029FBE0 = &D_8029FBE0;
+    ActionCommandStatus *d8029FBE0 = &gActionCommandStatus;
```

Comment:
Our pointer style is to do `ActionCommandStatus* d8...` with the pointer attached to the type

Reply:
this applies elsewhere, just mentioning it here once